### PR TITLE
integrated with the front end

### PR DESCRIPTION
**Pull request by harshavardhan-sureify**
image

### CODE REVIEW

File: src/components/admin/AllUsers.jsx [hunk 1]
```diff
@@ -60,6 +85,10 @@ const AllUsers = () => {
             .catch((err) => {
                 console.log(err);
             });
```
Comment (akhilarjun3-sureify) [inline src/components/admin/AllUsers.jsx:87]: reomove consolelogs

**Review by maniknr-sureify — APPROVED** [review]

File: src/components/admin/AllUsers.jsx (hunk 1 above)
Comment (harshavardhan-sureify) [reply src/components/admin/AllUsers.jsx]: removed

File: src/components/admin/AllUsers.jsx
[the remark this answers is not in the document]
Comment (harshavardhan-sureify) [reply src/components/admin/AllUsers.jsx:1]: already did it

**Review by akhilarjun3-sureify — APPROVED** [review]

File: src/constants.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const loginApi = GO_BASE_URL + "login";
 const adminDashboardApi = GO_BASE_URL + "admin/dashboard";
 const allUsers = PHP_BASE_URL + "viewusers";
 const userInfo = GO_BASE_URL + "user/userinfo";
+const userDelete = GO_BASE_URL+"admin/delete"
```
Comment (abhishekdundigalla-sureify) [inline src/constants.js:13]: instead of "userDelete", better rename it to "deleteUser"

Comment (harshavardhan-sureify) [reply src/constants.js:13]: renamed it

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Pull request merged**